app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -1,38 +1,36 @@
-- if current_user.admin? && @membership_application.status == 'Godkänd'
-  = f.label :membership_number, 'Medlemsnummer', class: 'required'
+- if current_user.admin? && @membership_application.status == t('membership_applications.accepted')
```

Comment:
Yes.  Good catch!
And apparently we don't have an integration test for this.  

_(I will be so glad when we have the states cleaned up and aren't checking hardcoded strings!)_